### PR TITLE
5e Spell Changes

### PR DESCRIPTION
**Pull request by Celestro**
The majority are duration spell changes to match their DnD 5e versions, excluding those that would exceed 1 hour. Other changes include:
- Changed Hail of Thorns to a Shout to apply a status to the character that applies the spell's effect on the next ranged weapon attack.
- Changed Feign Death to also grant a Dismiss spell for the caster.
- Changed Spirit Guardians to not only receive the `SPIRIT_GUARDIANS_DAMAGE_RECEIVED` status when initially casting the spell, since it's not supposed to deal its damage immediately to those in its area but also to increase its area radius/aura to 4.5m (believe it will round down to 4 so dunno if 5 would be more preferred).
- Changed Sanctuary so its status will actually force the Wisdom saving throw from all hostile creatures specifically in Turn Based mode (so it doesn't potentially aggro enemies that would be kind of too far away from the actual fight itself especially as I've received word that it technically applies on different floors).
- Changed Goodberry so it would not only grant 10 berries that heal for 1 HP similar to what you did originally, but it'll also grant a separate set of 10 berries for a creature with the Blessed Healer feature. When these are consumed, the creature will be healed the additional amount only if they don't have the Blessed Healer passive since it'll already apply to a Cleric with the passive when they eat a berry. Currently, it's only been accounted for for a 1st level spell however.

### CODE REVIEW

File: RAW/Public/RAW/Stats/Generated/Data/Spell_Target.txt [hunk 1]
```diff
@@ -713,3 +713,760 @@ using "Target_CallLightning_LightningBolt"
...
+data "CastEffect" "bcd66fb0-b0bc-41d0-abba-ad443d63dd72"
+data "TargetEffect" "4a3e84ac-6f7d-4759-8a94-5596071e2dcf"
+data "RitualCosts" "ActionPoint:1"
+
+new entry "Target_FeignDeath_Dismiss"
+type "SpellData"
+data "SpellType" "Target"
+using "Target_FeignDeath"
+data "SpellProperties" "RemoveStatus(FEIGN_DEATH);RemoveStatus(SELF,RAW_FEIGN_DEATH_DISMISS)"
+data "TargetConditions" "HasStatus('FEIGN_DEATH')"
```
Comment (FelipeRenault) [inline RAW/Public/RAW/Stats/Generated/Data/Spell_Target.txt:996]: Just to be extra careful and make sure only the caster of THAT Feign Death can cancel it 😅 
```suggestion
data "TargetConditions" "HasStatus('FEIGN_DEATH',context.Target,context.Source)"
```

File: RAW/Localization/English/RAW.xml [hunk 2]
```diff
@@ -100,7 +100,8 @@
     <content contentuid="h4507b659g8c44g4f43gaf5cgb83ac55eec62" version="1">A stabilized character no longer has to roll Death Saving Throws and will regain [1] after leaving combat.</content><!-- Stabilized -->
     <content contentuid="h518c02f3gda53g498dg95e8g21599aaae760" version="1">This summon can act because its Owner commanded it.</content><!-- Summon Resource Allowed -->
     <content contentuid="he3c1c67bgae0cg4968ga86egd2b1325c01b3" version="1">While inside the storm, this character can be target by a lightning bolt</content><!-- Call Lightning -->
-    
+    <content contentuid="a79d9187053947ab83babc46d76b8834" version="1">Next ranged weapon attack deals [1] to target and creatures within [2] of them.</content><!-- Hail of Thorns -->
```
Comment (FelipeRenault) [inline RAW/Localization/English/RAW.xml:103]: Can you make all UUIDs generated for localization files obey the format Larian uses?
Basically, they generate a normal UUID4, add `h` at the start and replace all `-` with `g` (for some reason)
```suggestion
    <content contentuid="ha79d9187g0539g47abg83bagbc46d76b8834" version="1">Next ranged weapon attack deals [1] to target and creatures within [2] of them.</content><!-- Hail of Thorns -->
```

**Review by FelipeRenault — COMMENTED** [review]
If you can, please insert all statuses that had their durations increased on `Public\RAW\Scripts\Game\RAW_Statuses.gameScript` so they're properly removed when the character takes a Short Rest (some are already there, leftovers from EA where I had some status duration increases)

**Comment by FelipeRenault** [comment]
Since I changed the structure of the mod, every file will now be conflicting.
Don't worry, I'll fix it myself

**Comment by Celestro** [comment]
Ah I gotcha.

**Pull request closed without merging**